drivers/scheduler/scheduler.go

```diff
@@ -347,6 +347,9 @@ type Driver interface {
 	// SetConfig sets connnection config (e.g. kubeconfig in case of k8s) for scheduler driver
 	SetConfig(configPath string) error
 
+	// SetGkeConfig sets connnection config (e.g. kubeconfig in case of k8s) for scheduler driver
```

Comment:
Change the doc string to indicate that this is for GKE clusters

Reply:
done